includes/admin/class-wp-job-manager-writepanels.php

```diff
@@ -631,15 +634,15 @@ public function save_job_listing_data( $post_id, $post ) {
 						delete_post_meta( $post_id, $key );
 					}
 				} else {
-					update_post_meta( $post_id, $key, date( 'Y-m-d', strtotime( sanitize_text_field( $_POST[ $key ] ) ) ) );
+					update_post_meta( $post_id, $key, date( 'Y-m-d', strtotime( sanitize_text_field( wp_unslash( $_POST[ $key ] ) ) ) ) );
 				}
 			} elseif ( '_job_author' === $key ) {
 				if ( empty( $_POST[ $key ] ) ) {
 					$_POST[ $key ] = 0;
 				}
 				$wpdb->update( $wpdb->posts, array( 'post_author' => $_POST[ $key ] > 0 ? absint( $_POST[ $key ] ) : 0 ), array( 'ID' => $post_id ) );
 			} elseif ( isset( $_POST[ $key ] ) ) {
-				update_post_meta( $post_id, $key, $_POST[ $key ] );
+				update_post_meta( $post_id, $key, wp_unslash( $_POST[ $key ] ) ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized -- Input sanitized in registered post meta config
```

Comment:
Maybe include a file reference, so someone can more easily go and find out what the sanitization is exactly?

```
// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized -- Input sanitized in registered post meta config - see /includes/foo/post-meta.php
```